.gitignore

```diff
@@ -1,33 +1,34 @@
-*~
-*.gch*
-*.obj
-*.o
-*.s
-*.exe
-Factor/factor
+*#*#
+*.*.marks
+*.RES
 *.a
+*.bak
 *.dll
-*.lib
+*.dylib
+*.exe
 *.exp
-*.res
-*.RES
+*.gch*
 *.image
-factor.image.fresh
-*.dylib
-factor.com
-*#*#
-.DS_Store
-.gdb_history
-*.*.marks
+*.lib
+*.o
+*.obj
+*.res
+*.s
+*.so
+*~
+.#*
 .*.swm
 .*.swn
 .*.swo
 .*.swp
+.DS_Store
+.gdb_history
+/factor
```

Comment:
Other option here would just be too make it `factor`, which removes the need for the following line, but I wasn't sure that was actually desirable.

Reply:
I think that makes sense.

Reply:
Oh wait.

Reply:
I'm not sure I understand how that relates to this diff; `factor` in `.gitignore` won't match e.g. `zealot.factor`, just any file named exactly `factor` at any point in the directory hierarchy. If you want to ban anything with the word `factor` in it, it'd need to be `*factor*`. (Which I'm assuming we don't want, but just saying.)

If you're saying that zealot negates the need for this rule entirely, I don't think that's correct, either; running `./build.sh update` was why I proposed adding these rules.

I might be missing something here, though; zealot's new, and I may be misunderstanding your point.